socorro/external/es/super_search_fields.py

```diff
@@ -109,23 +118,12 @@ def add_doc_values(value):
             add_doc_values(field)
 
 
-class SuperSearchFields(object):
-    # Defining some filters that need to be considered as lists.
-    filters = [
-        ('form_field_choices', None, ['list', 'str']),
-        ('permissions_needed', None, ['list', 'str']),
-    ]
+class SuperSearchFieldsData(object):
```

Comment:
Having a data-only class makes it easier for other things to use the fields without having to configure an Elasticsearch `ConnectionContext`.